### PR TITLE
Update Home-User-Meta.js

### PR DESCRIPTION
**Pull request by dansegliode**
added function that only returns link when a profile is loaded

### CODE REVIEW

File: src/components/Home-User-Meta.js [hunk 1]
```diff
@@ -121,7 +127,7 @@ class HomeUserMeta extends Component {
           { this.userIsHirable() }
         </div>
         <div className="User-Meta-profile">
-          <img className="User-Meta-profile-picture" src={ this.userImg() } alt="user" />
+          <a href={ this.userImgUrl() } target="_blank"><img className="User-Meta-profile-picture" src={ this.userImg() } alt="user" /></a>
```
Comment (TheWillG) [inline src/components/Home-User-Meta.js]: Hi @dansegliode is the `className` being used? If not please remove

**Comment by theblindprophet** [comment]
There is already a PR for this. https://github.com/theblindprophet/github-profile-summary/pull/39

File: src/components/Home-User-Meta.js (hunk 1 above)
Comment (dansegliode) [reply src/components/Home-User-Meta.js]: yes, it was there before

Comment (TheWillG) [reply src/components/Home-User-Meta.js]: Sorry I did not see this :)

Comment (theblindprophet) [reply src/components/Home-User-Meta.js]: Read comment below.

**Comment by theblindprophet** [comment]
@dansegliode The `<a>` tags need to be conditionally added is there is a user. e.g.

```
if (this.props.userData && this.props.userData.url) {
  return (
    <a href={ this.userUrl() } target="_blank"><img className="User-Meta-profile-picture" src={ this.userImg() } alt="user" /></a>
  );
}
return (<img className="User-Meta-profile-picture" src={ this.userImg() } alt="user" />);
```

**Pull request merged**